src/dmd/target.d

```diff
@@ -1240,6 +1242,67 @@ struct TargetC
             assert(0);
     }
 
+    /**
+     * Builds the C `__builtin_va_list` type for target.
+     * NOTE: This is used for sources compiled in ImportC mode.
+     * Returns:
+     *      `Type` that represents `__builtin_va_list`.
+     */
+    extern (C++) Type va_listType() const
+    {
+        import dmd.mtype : TypePointer;
+
+        if (target.os == Target.OS.Windows)
+        {
+            return new TypePointer(Type.tchar);
+        }
+        else if (target.os & Target.OS.Posix)
+        {
+            if (target.is64bit)
```

Comment:
invert this to get an early return

Reply:
I think I'll move this bit to master and just do the minimum dmd needs to get away with it.

I think it'd be better to change the signature to `void(ref Dsymbols)` so then this just pushes out all symbols that get predefined for the target (in gdc, I can dump all gcc builtins).